File: webservices/CORE/Usuarios.php
<?php


class Usuarios
{
    public $idusuarios;
    public $usuario;
    public $clave;
    public $rol;
    public $persona;
    public $estado;

    /**
     * @return mixed
     */
    public function getIdusuarios()
    {
        return $this->idusuarios;
    }

    /**
     * @param mixed $idusuarios
     */
    public function setIdusuarios($idusuarios)
    {
        $this->idusuarios = $idusuarios;
    }

    /**
     * @return mixed
     */
    public function getUsuario()
    {
        return $this->usuario;
    }

    /**
     * @param mixed $usuario
     */
    public function setUsuario($usuario)
    {
        $this->usuario = $usuario;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }


    /**
     * Usuarios constructor.
     * @param $isusuario
     * @param $nombre
     * @param $clave
     * @param $rol
     * @param $persona
     */
    public function __construct($isusuario, $nombre, $clave, $rol, $persona)
    {
        $this->idusuarios = $isusuario;
        $this->usuario = $nombre;
        $this->clave = $clave;
        $this->rol = $rol;
        $this->persona = $persona;
    }

    /**
     * @return mixed
     */
    public function getIdusuario()
    {
        return $this->idusuario;
    }

    /**
     * @param mixed $isusuario
     */
    public function setIdusuario($isusuario)
    {
        $this->idusuario = $isusuario;
    }

    /**
     * @return mixed
     */
    public function getNombre()
    {
        return $this->nombre;
    }

    /**
     * @param mixed $nombre
     */
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;
    }

    /**
     * @return mixed
     */
    public function getClave()
    {
        return $this->clave;
    }

    /**
     * @param mixed $clave
     */
    public function setClave($clave)
    {
        $this->clave = $clave;
    }

    /**
     * @return mixed
     */
    public function getRol()
    {
        return $this->rol;
    }

    /**
     * @param mixed $rol
     */
    public function setRol($rol)
    {
        $this->rol = $rol;
    }

    /**
     * @return mixed
     */
    public function getPersona()
    {
        return $this->persona;
    }

    /**
     * @param mixed $persona
     */
    public function setPersona($persona)
    {
        $this->persona = $persona;
    }



}

File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Usuarios.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Usuarios implements Serializable {
    public int idusuario;
    public String usuario;
    public String clave;
    public int rol;
    public int persona;
    private int idbase;



    public Usuarios(int idusuario, String usuario, String clave, int rol, int persona) {
        this.idusuario = idusuario;
        this.usuario = usuario;
        this.clave = clave;
        this.rol = rol;
        this.persona = persona;
    }

    public Usuarios() {

    }
    public int getIdbase() {
        return idbase;
    }

    public void setIdbase(int idbase) {
        this.idbase = idbase;
    }

    public int getIdusuario() {
        return idusuario;
    }

    public void setIdusuario(int idusuario) {
        this.idusuario = idusuario;
    }

    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public String getClave() {
        return clave;
    }

    public void setClave(String clave) {
        this.clave = clave;
    }

    public int getRol() {
        return rol;
    }

    public void setRol(int rol) {
        this.rol = rol;
    }

    public int getPersona() {
        return persona;
    }

    public void setPersona(int persona) {
        this.persona = persona;
    }
}
